# Origin, evolution and classification of type-3 copper proteins: lineage-specific gene expansions and losses across the Metazoa - Supplementary Data 

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Origin, evolution and classification of type-3 copper proteins: lineage-specific gene expansions and losses across the Metazoa.** Felipe Aguilera, Carmel McDougall, Bernard M. Degnan. *BMC Evolutionary Biology* **13**:96 (2013). https://doi.org/10.1186/1471-2148-13-96.

**Abstract**

**Background:** Tyrosinases, tyrosinase-related proteins, catechol oxidases and hemocyanins comprise the type-3 copper protein family and are involved in a variety of biological processes, including pigment formation, innate immunity and oxygen transport. Although this family is present in the three domains of life, its origin and early evolution are not well understood. Previous analyses of type-3 copper proteins largely have focussed on specific animal and plant phyla. **Results:** Here, we combine genomic, phylogenetic and structural analyses to show that the original type-3 copper protein possessed a signal peptide and may have been secreted (we designate proteins of this type the α subclass). This ancestral type-3 copper protein gene underwent two duplication events, the first prior to the divergence of the unikont eukaryotic lineages and the second before the diversification of animals. The former duplication gave rise to a cytosolic form (β) and the latter to a membrane-bound form (γ). Structural comparisons reveal that the active site of α and γ forms are covered by aliphatic amino acids, and the β form has a highly conserved aromatic residue in this position. The subsequent evolution of this gene family in modern lineages of multicellular eukaryotes is typified by the loss of one or more of these three subclasses and the lineage-specific expansion of one or both of the remaining subclasses. **Conclusions:** The diversity of type-3 copper proteins in animals and other eukaryotes is consistent with two ancient gene duplication events leading to α, β and γ subclasses, followed by the differential loss and expansion of one or more of these subclasses in specific kingdoms and phyla. This has led to many lineage-specific type-3 copper protein repertoires and in some cases the independent evolution of functionally-classified tyrosinases and hemocyanins. For example, the oxygen-carrying hemocyanins in arthropods evolved from a β-subclass tyrosinase, whilst hemocyanins in molluscs and urochordates evolved independently from an α-subclass tyrosinase. Minor conformational changes at the active site of α, β and γ forms can produce type-3 copper proteins with capacities to either carry oxygen (hemocyanins), oxidize diphenols (catechol oxidase) or o-hydroxylate monophenols (tyrosinases) and appear to underlie some functional convergences.

## Author contact

- [Felipe Aguilera](mailto:f.aguilera@uq.edu.au) (first author - PhD student)
- [Carmel McDougall](mailto:c.mcdougall@uq.edu.au) (senior author - PostDoc)
- [Bernie Degnan](b.degnan@uq.edu.au) (corresponding author)
- Lab website: https://www.aguileralab.com

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Dataset.xlsx](https://github.com/faguil/Type3-Copper-Protein-Evolution/blob/main/01-Dataset.xlsx) contains the list of accession numbers, genome localisation and protein nomenclature used in this study.

[02-Phylogenetic-tress-alpha-subclass-copper-proteins.pdf](https://github.com/faguil/Type3-Copper-Protein-Evolution/blob/main/02-Phylogenetic-tress-alpha-subclass-copper-proteins.pdf) contains the phylogenetic trees of the α-subclass copper proteins based on Neighbor-Joining (NJ), Maximum-Likelihood (ML), and Bayesian Inference (BI) phylogenetic reconstruction methods.

[03-Phylogenetic-tress-beta-subclass-copper-proteins.pdf](https://github.com/faguil/Type3-Copper-Protein-Evolution/blob/main/03-Phylogenetic-tress-beta-subclass-copper-proteins.pdf) contains the phylogenetic trees analyses of the β-subclass copper proteins based on Neighbor-Joining (NJ), Maximum-Likelihood (ML), and Bayesian Inference (BI) phylogenetic reconstruction methods.

[04-Phylogenetic-tress-gamma-subclass-copper-proteins.pdf](https://github.com/faguil/Type3-Copper-Protein-Evolution/blob/main/04-Phylogenetic-tress-gamma-subclass-copper-proteins.pdf) contains the phylogenetic trees of the γ-subclass copper proteins based on Neighbor-Joining (NJ), Maximum-Likelihood (ML), and Bayesian Inference (BI) phylogenetic reconstruction methods. 

[05-treeBase-aligmnent-files](https://github.com/faguil/Type3-Copper-Protein-Evolution/tree/main/05-treeBase-alignment-files) This folder contains the alignment files used in constructing the phylogenetic trees for this study. More detailed information about these files can be found in the TreeBase database at the following link: https://treebase.org/treebase-web/search/study/summary.html?id=13985.

[06-treeBase-phylogenetic-tree-files](https://github.com/faguil/Type3-Copper-Protein-Evolution/tree/main/06-treeBase-phylogenetic-trees-files)) This folder contains all the phylogenetic trees generated in this study using the Neighbor-Joining (NJ), Maximum-Likelihood (ML), and Bayesian Inference (BI) methods. More detailed information about these files is available in the TreeBase database at the following link: https://treebase.org/treebase-web/search/study/summary.html?id=13985.

[07-Lineage-specific-expansions-copper-proteins.pdf](https://github.com/faguil/Type3-Copper-Protein-Evolution/blob/main/07-Lineage-specific-expansions-copper-proteins.pdf) contains lineage-specific expansion of type-3 copper gene subclasses and their physical linkage.
